### ISSUE TITLE
Could PyOD be implemented in the CV datasets in the future?

### ISSUE THREAD

**Issue by jmq19950824** [yzhao062/pyod#334]
I appreciate that pyod now have added some cutting-edge techniques in anomaly detection domain like SO-GAAL, MO-GAAL and Deep SVDD.
**Could these models also be implemented in the computer vision datasets (like MNIST or CIFAR10) in the future?** Besides, the improved version of Deep SVDD——[Deep SAD](https://github.com/lukasruff/Deep-SAD-PyTorch) may be added in pyod.
Thanks!

**Comment by yzhao062** [comment]
Thanks for the note. For the CV data, if you prepare them in the dataloader and have the transformation, some models may directly handle them. Yeah, I am aware of deepSAD--just lack the bandwidth to implement. Feel free to contribute if you want :)

**Comment by jmq19950824** [comment]
> Thanks for the note. For the CV data, if you prepare them in the dataloader and have the transformation, some models may directly handle them. Yeah, I am aware of deepSAD--just lack the bandwidth to implement. Feel free to contribute if you want :)

Thanks for the reply! Does the transformation mean that we should flatten the CV images to the tabular form? (e.g., transform the original 1000x32x32 image to a 1000x1024 tabular data). However, implementing the convolutional structure to the existing pyod models may be a better choice.

**Comment by yzhao062** [comment]
Yes. This is only for the existing models...transforming to tabular data is needed. I am not sure when will more dl models, e.g., with conv layers, will be released. If you happen to implement some, feel free to contribute :)

**Comment by jmq19950824** [comment]
> Yes. This is only for the existing models...transforming to tabular data is needed. I am not sure when will more dl models, e.g., with conv layers, will be released. If you happen to implement some, feel free to contribute :)

Thanks. Actually,  I have tried some anomaly detection models implemented with convolutional layers, e.g., DeepSAD (with LeNet backbone as in the original paper) and GANomaly (with DCGAN backbone), maybe I would contribute to the pyod next time :)))
